src/main/java/nl/tudelft/dnainator/ui/DNAinator.java

```diff
@@ -24,20 +32,57 @@
 	private static final int MIN_WIDTH = 300;
 	private static final int MIN_HEIGHT = 150;
 
+	private Stage stage;
+	private BooleanProperty ready = new SimpleBooleanProperty(false);
+
+	private void loadDB() {
+		Task<Graph> task = new Task<Graph>() {
+			@Override
+			protected Graph call() throws Exception {
```

Comment:
The overridden method throws it by default, we do not actually throw an exception.